test/projectOverviewResponse.test.js

```diff
@@ -0,0 +1,19 @@
+const { expect } = require('chai');
+const ProjectOverviewResponse = require('../lib/actions/projectOverviewResponse.js');
+
+describe('Project Overview Response', () => {
+  it('returns a formatted list of links', () => {
+    const arrayOfLinks = [
+      { label: 'facebook', link: 'facebook.com' },
+      { label: 'google', link: 'google.com' },
+    ];
+    const expectedResult = '1. facebook facebook.com\n2. google google.com';
+    let replyResponse = '';
+    function replyMock(message, formattedResponse) {
+      replyResponse = formattedResponse;
+    }
+    const responseInstance = new ProjectOverviewResponse(arrayOfLinks);
+    responseInstance.sendToBot({ reply: replyMock });
+    expect(replyResponse).to.equal(expectedResult);
```

Comment:
Great test! 🏆

Reply:
What happens when the list of links are empty? That is, if someone did `/project` before `/project add` ever happens, what will the output be? What would make sense for it to say?

Reply:
hmm, makes sense. Will do.